velox/functions/lib/TimeUtils.h

```diff
@@ -26,12 +26,30 @@ inline constexpr int64_t kSecondsInDay = 86'400;
 inline constexpr int64_t kDaysInWeek = 7;
 extern const folly::F14FastMap<std::string, int8_t> kDayOfWeekNames;
 
+// Format timezone to make it compatible with date::locate_zone.
+// For example, converts "GMT+8" to "Etc/GMT-8".
+// Here is the list of IANA timezone names:
+// https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+FOLLY_ALWAYS_INLINE std::string formatTimezone(const std::string& timezone) {
+  if (timezone.find("GMT") == 0 && timezone.size() > 4) {
+    if (timezone[3] != '+' && timezone[3] != '-') {
+      return timezone;
+    }
+    std::string prefix = timezone[3] == '+' ? "Etc/GMT-" : "Etc/GMT+";
```

Comment:
This format will be more efficient and precise. 
```
if (timezone[3] == '+') {
      return "Etc/GMT-" + timezone.substr(4);
    } else if (timezone[3] == '-') {
      return "Etc/GMT+" + timezone.substr(4);
    } else {
      return timezone;
    }
```

Reply:
@jinchengchenghh, yes, thanks for your suggestion! Just updated.